src/components/sections/items/ToggleSectionItem.tsx

```diff
@@ -61,13 +64,18 @@ export function ToggleSectionItem({
       style={topContainer}
       accessible={true}
       accessibilityRole="switch"
-      accessibilityState={{checked: checked}}
+      accessibilityState={{checked: checked, disabled: disabled}}
       accessibilityActions={[{name: 'activate'}]}
       onAccessibilityAction={() => onChange(!checked)}
```

Comment:
Would maybe handle the `disabled` flag here instead:
`onAccessibilityAction={disabled ? undefined : () => onChange(!checked)}`